android/app/src/main/res/values-de/strings.xml

```diff
@@ -827,12 +827,15 @@
 	<string name="type.amenity.parking.private">Privatparkplatz</string>
 	<string name="type.amenity.parking.park_and_ride">P+R-Parkplatz</string>
 	<string name="type.amenity.parking.underground">Tiefgarage</string>
-	<string name="type.amenity.parking.street_side">Parkplatz</string>
-	<string name="type.amenity.parking.fee.street_side">Parkplatz</string>
-	<string name="type.amenity.parking.lane">Parkplatz</string>
-	<string name="type.amenity.parking.fee.lane">Parkplatz</string>
+	<string name="type.amenity.parking.fee.underground">Bezahlter tiefgaragenplatz</string>
+	<string name="type.amenity.parking.underground.private">Private tiefgarage</string>
+	<string name="type.amenity.parking.private.underground">Private tiefgarage</string>
+	<string name="type.amenity.parking.street_side">Parken an der straße</string>
+	<string name="type.amenity.parking.fee.street_side">Bezahltes parken am straßenrand</string>
+	<string name="type.amenity.parking.street_side.private">Privat straße seite parken</string>
+	<string name="type.amenity.parking.private.street_side">Privat straße seite parken</string>
```

Comment:
```suggestion
	<string name="type.amenity.parking.fee.underground">Gebührenpflichtige Tiefgarage</string>
	<string name="type.amenity.parking.underground.private">Private Tiefgarage</string>
	<string name="type.amenity.parking.private.underground">Private Tiefgarage</string>
	<string name="type.amenity.parking.street_side">Parkbucht</string>
	<string name="type.amenity.parking.fee.street_side">Gebührenpflichtige Parkbucht</string>
	<string name="type.amenity.parking.street_side.private">Private Parkbucht</string>
	<string name="type.amenity.parking.private.street_side">Private Parkbucht</string>
```

Reply:
_Gebührenpflichtig_ is a mouthful.
Maybe _Kostenpflichtig_ (which is still long)?

Reply:
Yeah its tricky, german words...

Reply:
But I agree to use the term `Kostenpflichtig` as it is also used for `type.amenity.parking.fee`
Thanks for the hint :+1: